package/MDAnalysis/coordinates/PDB.py

```diff
@@ -574,9 +578,6 @@ def _write_pdb_title(self):
                        "".format(self.start, self.remarks))
 
     def _write_pdb_header(self):
-        if not self.obj or not hasattr(self.obj, 'universe'):
-            self._write_pdb_title(self)
```

Comment:
There's already a check before this method is called (`_update_frame`) so this check was redundant.